pkg/pillar/controllerconn/send.go

```diff
@@ -822,6 +850,21 @@ func (c *Client) SendOnIntf(ctx context.Context, destURL string, intf string,
 			if err != nil {
 				errorLog(err.Error())
 			} else {
+				if sink != nil {
+					jsonPath := fmt.Sprintf("%s/nettrace_%s.json", opts.NetTraceFolder, nettraceSessionUUID)
+					if err := sink.ExportToJSON(jsonPath, netTrace.NetTrace); err != nil {
```

Comment:
How did we choose this approach: Writing to a DB first, then dump in JSON? I'm not saying it's not right or bad, I'm just trying to understand the motivation.